Shapes/shapes/rr/graph_map.ttl

```diff
@@ -0,0 +1,85 @@
+###############################################################################
+# R2RML Graph Map and graph shortcut shape                                    #
+# Author: Dylan Van Assche                                                    #
+# Copyright IDLab - UGent - IMEC (2020)                                       #
+###############################################################################
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix schema: <http://schema.org/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rml: <http://semweb.mmlab.be/ns/rml#> .
+@prefix rr: <http://www.w3.org/ns/r2rml#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> . 
+
+schema:RRGraphMapShape
+    a sh:NodeShape ;
+    sh:closed "true"^^xsd:boolean ;
+    sh:ignoredProperties (rdf:type) ;
+    sh:name "rr:graphMap" ;
+    sh:description """
+    Represents a graph map.
+    """ ;
+    sh:message """
+    rr:GraphMap must specify an rr:template or rr:constant with the IRI of the 
+    Named Graph.
```

Comment:
does this come from the spec? bc in theory a graph can also be generated with an `rr:column` or `rml:reference` if they are combined with `rr:IRI` term type, no?

Reply:
I think here we actually need to add the term type? I think you do so ine case of the object map. 

I think the current best reference for all possible combinations of term maps and types is the table from my book chaper

Reply:
Agreed.

Curious that [R2RML restricts](https://www.w3.org/TR/r2rml/#named-graphs) graph names to only be IRI's, since the [RDF rec](https://www.w3.org/TR/rdf11-concepts/#section-dataset) allows both IRI's and blank nodes.

Reply:
> Possible target graphs are the unnamed default graph, and the IRI-named named graphs

Indeed, it seems they implicitly limited it to IRIs only. It's not mentioned that the `rr:termType` must be `rr:IRI`.

Speaking of Term Maps, Graph Maps are Term Maps so in theory they allow *any* type:

> Graph maps are themselves term maps. 

So we should in the new spec also make this more clear that you cannot have an RDF Literal here so that we either need `rr:IRI` or `rr:BlankNode` here.